cmd/minishift/cmd/status.go

```diff
@@ -43,6 +44,14 @@ DiskUsage:  {{.DiskUsage}}
 CacheUsage: {{.CacheUsage}} (used by oc binary, ISO or cached images)
 `
 
+var statusFormatWithRegistration = `Minishift:  {{.MinishiftStatus}}
+Profile:    {{.ProfileName}}
+OpenShift:  {{.ClusterStatus}}
+DiskUsage:  {{.DiskUsage}}
+CacheUsage: {{.CacheUsage}} (used by oc binary, ISO or cached images)
+RHSM: 	    {{.Registration}}
```

Comment:
@anjannath looks like there is a space needed in between `RHSM:` and `{{.Registration}}` to make it same as other variables in the template.

Reply:
@praveenkumar It appears properly aligned in my editor locally, could you please check locally once, if it is not aligned i'll modify then.

Reply:
@anjannath nvm, But why you used a different variable for registration purpose? I am hoping to have it part of `statusFormat` variable only and ISO who doesn't support registration can have `Not Applicable` as the output.

Reply:
@praveenkumar I thought about it too, but this looks more cleaner, at the cost of an extra string variable so..  :-)